liveman_api/lib/liveman_api.ex

```diff
@@ -5,23 +5,21 @@ defmodule LivemanApi do
 
   @timeout 30_000
 
-  def get(base_url, url, query \\ %{}) do
+  def get(base_url, url, query \\ %{}, headers \\ []) do
     base_url
-    |> client()
+    |> client(headers)
     |> Tesla.get(url, query: Map.to_list(query))
     |> handle_response()
   end
 
-  def post(base_url, url, body \\ %{}) do
+  def post(base_url, url, body \\ %{}, headers \\ []) do
     base_url
-    |> client()
+    |> client(headers)
     |> Tesla.post(url, body)
```

Comment:
We can also pass the headers here, right? 🤔  Is it gonna merge to the header of the client? I guess it is 💭 

```
|> Tesla.post(url, body, headers: headers)
```

Reply:
Yup, I've moved the headers to the method 22bb3ea3db940b0838e64b8577cabca5e8465294 😄